applicationLibrary/molecularDynamics/tests/templateInstatiations/AutoPasInstantiations.cpp

```diff
@@ -13,5 +13,5 @@
 template class autopas::AutoPas<Molecule>;
 template bool autopas::AutoPas<Molecule>::iteratePairwise(
     mdLib::LJFunctor<Molecule, /* shifting */ false, /*mixing*/ false, autopas::FunctorN3Modes::Both,
-                     /*globals*/ false, /*relevantForTuning*/ true> *);
+                     /*globals*/ false, /*countFLOPs*/ false, /*relevantForTuning*/ true> *);
```

Comment:
```suggestion
// clang-format off
template bool autopas::AutoPas<Molecule>::iteratePairwise(
    mdLib::LJFunctor<Molecule,
                     /* shifting */           false,
                     /* mixing */             false,
                     autopas::FunctorN3Modes::Both,
                     /* globals */            false,
                     /* countFLOPs */         false,
                     /* relevantForTuning */  true> *);
// clang-format on
```
Just some overdue reformatting.

Reply:
Done in 7fa81126